ext/storage_sources/gcp_store/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+project(gcp_store CXX)
+
+find_package(google_cloud_cpp_common CONFIG REQUIRED)
+
```

Comment:
I think we can remove the blank lines between the `find_package` commands.

Reply:
addressed here: [e7cd853](https://github.com/wiredtiger/wiredtiger/pull/8688/commits/e7cd853492f5b96a6798679c3db140b75ce568eb)